ADMIN REDESIGN: redesign the Event Management's Action Items tab for the talawa Admin portal

Anvita0305 opened the pull request:
**What kind of change does this PR introduce?**
Redesign the screen

**Issue Number:**

Fixes #1751 

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**
Screen
[Events Action Items Updated Screen.webm](https://github.com/PalisadoesFoundation/talawa-admin/assets/78889572/13aab14e-19f5-4d6d-b6a2-041860dad55b)

Tests
![Screenshot from 2024-04-09 20-04-08](https://github.com/PalisadoesFoundation/talawa-admin/assets/78889572/80304d40-8826-4a2f-a929-14a58cc056ed)
a1b8c3958)


**If relevant, did you update the documentation?**
N/A

**Summary**
Redesigned the screen according to the figma designs provided.

**Does this PR introduce a breaking change?**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

palisadoes commented:
Please try to get the test code coverage higher

palisadoes reviewed (changes requested):
When you delete an action item, the modal remains. It should disappear exposing the background page

Anvita0305 commented:
Sure! Will add tests for missing lines and update delete modal

pranshugupta54 commented:
When you clicked on "Manage Actions" for Teresa, why was there a need to select the Assignee again? It should automatically get the selected user

Anvita0305 commented:
Hello, 
1. I have added the tests
2. Updated working of delete modal
Video:   [Delete Modal.webm](https://github.com/PalisadoesFoundation/talawa-admin/assets/78889572/34e1849f-5232-433b-82bd-74274ef0656a)

palisadoes commented:
Almost there:

1. On the "Manage Actions" Modal
    1. "Edit Action Item", should be just "Save"
    2. There is no option to edit the "pre-completion notes". This should be called "Notes" in all cases as it's more intuitive

        ![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/16875803/f33cc1c8-54a3-4ff3-90d5-7a4524b163f4)

1. On the "Action Items Detail" modal
    1. "Pre Completion Notes" should be just "Notes". It's more intuitive and it's also what is used in the heading

        ![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/16875803/0490b027-49b3-4535-bbe4-5b4ec78052d5)

Anvita0305 commented:
Updated Screens:
1. 
![Screenshot from 2024-04-12 09-05-39](https://github.com/PalisadoesFoundation/talawa-admin/assets/78889572/d403a832-2268-4541-9fe3-9de5ff8e3a48)

2.
![Screenshot from 2024-04-12 09-02-35](https://github.com/PalisadoesFoundation/talawa-admin/assets/78889572/abbc9218-4169-4964-820c-4ce84c6ab331)

pranshugupta54 commented:
it would look much better if we keep the buttons **left** aligned and button names as "Save Item" and "Delete Item"


Reference from event design:
<img width="300" alt="image" src="https://github.com/PalisadoesFoundation/talawa-admin/assets/76090263/aa892a6d-c7b2-4018-907a-e62f36fdb103">
